docs/Installing.asciidoc

```diff
@@ -311,6 +310,22 @@ NFS clients are where openQA workers are running. Run following command:
 mount -t nfs openQA-webUI-host:/var/lib/openqa/share /var/lib/openqa/share
 --------------------------------------------------------------------------------
 
+Needle Caching  
+~~~~~~~~~~~~~~~
+
+If you your network is slow or you experience long time to load needles, you
+might want to consider needle caching, to do this a directory
++/var/lib/openqa/cache+ must be created, and right permissions given to the
+'geekotest' user.
```

Comment:
If your network is slow or you experience long loading time for needles you
might want to consider needle caching. To use needle caching a directory
++/var/lib/openqa/cache+ must be created and and right permissions must given to the
+'geekotest' user.